types/lazy.js/index.d.ts

```diff
@@ -18,8 +16,6 @@ declare namespace LazyJS {
 
         repeat<T>(value: T, count?: number): GeneratedSequence<T>;
 
-        on<T>(eventType: string): Sequence<T>;
```

Comment:
Was this intentionally removed?

Reply:
Yes, it doesn't exist in lazy.js 0.5.1. Maybe it existed in earlier versions.